From 30c5a951a849f835acc9f11ea4ca4e1d3380b7eb Mon Sep 17 00:00:00 2001
From: Robin Ebert <ebertrobin2002@gmail.com>
Date: Wed, 25 Jan 2023 17:31:46 +0100
Subject: [PATCH] Destroy all created wayland object in terminateDisplay

---
 src/wayland-egldisplay.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/wayland-egldisplay.c b/src/wayland-egldisplay.c
index 16a948d..1f30a76 100644
--- a/src/wayland-egldisplay.c
+++ b/src/wayland-egldisplay.c
@@ -628,6 +628,10 @@ static EGLBoolean terminateDisplay(WlEglDisplay *display, EGLBoolean globalTeard
             wl_eglstream_display_destroy(display->wlStreamDpy);
             display->wlStreamDpy = NULL;
         }
+        if (display->wlStreamCtl) {
+            wl_eglstream_controller_destroy(display->wlStreamCtl);
+            display->wlStreamCtl = NULL;
+        }
         if (display->wpPresentation) {
             wp_presentation_destroy(display->wpPresentation);
             display->wpPresentation = NULL;
