diff -Naur egl-wayland-1.1.4/wayland-egl/wayland-egl-ext.h egl-wayland-1.1.4.tpg/wayland-egl/wayland-egl-ext.h
--- egl-wayland-1.1.4/wayland-egl/wayland-egl-ext.h	2019-09-12 23:42:00.000000000 +0000
+++ egl-wayland-1.1.4.tpg/wayland-egl/wayland-egl-ext.h	2020-02-25 15:20:33.349067910 +0000
@@ -26,6 +26,7 @@
 #ifndef EGL_WL_wayland_eglstream
 #define EGL_WL_wayland_eglstream 1
 #define EGL_WAYLAND_EGLSTREAM_WL              0x334B
+#define EGL_WAYLAND_Y_INVERTED_WL             0x31DB /* eglQueryWaylandBufferWL attribute */
 #endif /* EGL_WL_wayland_eglstream */
 
 #ifndef EGL_NV_stream_fifo_synchronous
